src/js/out/modules/node/fs.js

```diff
@@ -1,3 +1,4 @@
+var {EventEmitter } = import.meta.require("node:stream");
```

Comment:
This is not correct and I'm not surprised it works

Reply:
You mean the formatting?